dbms/tests/queries/0_stateless/00829_bitmap_function.reference

```diff
@@ -34,6 +34,10 @@
 [6,8,10]
 10
 [1,3,5,6,8,10,11,13,14,15]
+tag1	[0,1,2,3,4,5,6,7,8,9]	[]	[]	[0,1,2,3,4,5,6,7,8,9]
+tag2	[0,1,2,3,4,5,6,7,8,9]	[0]	[2]	[1,2,3,4,5,6,7,8,9]
+tag3	[0,1,2,3,4,5,6,7,8,9]	[0,7]	[3,101]	[1,2,3,4,5,6,8,9,101]
+tag4	[0,1,2,3,4,5,6,7,8,9]	[5,999,2]	[2,888,20]	[0,1,3,4,6,7,8,9,20]
```

Comment:
The result here is a bit surprising, especially since result depends on array ordering, might be worth explicit mention in docs.

Reply:
Added a note in doc.